port_ocean/core/handlers/entity_processor/jq_entity_processor.py

```diff
@@ -95,21 +96,26 @@ async def _search_as_bool(self, data: dict[str, Any], pattern: str) -> bool:
         )
 
     async def _search_as_object(
-        self, data: dict[str, Any], obj: dict[str, Any]
+        self,
+        data: dict[str, Any],
+        obj: dict[str, Any],
+        misconfigurations: dict[str, str] | None = None,
     ) -> dict[str, Any | None]:
         search_tasks: dict[
```

Comment:
so we are not returning it? only performing a manipulation of the object if passed?

Reply:
maybe worth initializing one if not passed from outside?